<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="classtreeWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>141</width>
      <height>301</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>用户列表</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>班级一</string>
     </property>
     <item>
      <property name="text">
       <string>小明</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>小华</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>小书</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>班级二</string>
     </property>
     <item>
      <property name="text">
       <string>小朱</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>小王</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>小歆</string>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QTreeWidget" name="datatreeWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>信息</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>1</string>
     </property>
     <property name="text">
      <string>喜欢玩</string>
     </property>
     <property name="text">
      <string>学习</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>380</y>
      <width>571</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_makenum">
       <property name="text">
        <string>生成随机数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Dlenum">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_changenum">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_HttpStatus">
       <property name="text">
        <string>打开Http</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>首页</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
